app/models/organizations/hearings_supervisor.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class HearingsSupervisor < Organization
```

Comment:
```suggestion
class HearingsSupervisors < Organization
```
Preface: The way you have this class name written is exactly how I wrote it in the AC... However, I think it would make more sense to have it be plural. What do you think?